js/app.js

```diff
@@ -2,147 +2,184 @@
 
 //Adds score variable to count for correct answers
 let score = 0;
 
-//Question 1:
-let game = prompt('Would you like to play a game? All responses should be \'yes\' or \'no\'.').toLowerCase();
-
-if (game === 'yes') {
-//console.log('Eggselent');
-  alert('Eggselent');
-  score++;
-} else if (game === 'no') {
-//console.log('Too bad! We\'re playing anyway!');
-  alert('Too bad! We\'re playing anyway!');
-} else {
-//console.log('Hmm, I don\'t understand. Let\'s play anyways.');
-  alert('Hmm, I don\'t understand. Let\'s play anyways.');
-}
-
-//Question 2:
-let military = prompt('Was I in the Navy?').toLowerCase();
-
-if (military === 'yes') {
-//console.log('Nope. I was in the Air Force');
-  alert('Nope. I was in the Air Force.');
-} else if (military === 'no') {
-  //console.log('That\'s right! I was in the Air Force.');
-  alert('That\'s right! I was in the Air Force.');
-  score++;
-} else {
-  //console.log('I see you\'re playing hardball. Let\'s move on.');
-  alert('I see you\'re playing hardball. Let\'s move on.');
-}
-
-//Question 3:
-let hometown = prompt('Am I from Western New York?').toLowerCase();
-
-if (hometown === 'yes') {
-  //console.log('That\'s right! From the southern shores of Lake Ontario.');
-  alert('That\'s right! From the southern shores of Lake Ontario.');
-  score++;
-} else if (hometown === 'no') {
-  //console.log('WHAT?! Do you even know me at all? Of course I am!');
-  alert('WHAT?! Do you even know me at all? Of course I am!');
-} else {
-  //console.log('Don\'t like to follow the rules? That\'s okay.');
-  alert('Don\'t like to follow the rules? That\'s okay.');
-}
-
-//Question 4:
-let peach = prompt('Did I play in peach orchards as a kid?').toLowerCase();
-
-if (peach === 'yes') {
-  //console.log('I sure did. The climate of the great lakes helps to enable fruit tree orchards in such a cold climate.');
-  alert('I sure did. The climate of the great lakes helps to enable fruit tree orchards in such a cold climate.');
-  score++;
-} else if (peach === 'no') {
-  //console.log('Uhm, I actually did. Awkward.');
-  alert('Uhm, I actually did. Awkward.');
-} else {
-  //console.log('At a loss for words, eh?');
-  alert('At a loss for words, eh?');
-}
-
-//Question 5:
-let daughters = prompt('Do I have three daughters?').toLowerCase();
-
-if (daughters === 'yes') {
-  //console.log('Not yet, but you never know in the future! The answer is two.');
-  alert('Not yet, but you never know in the future! The answer is two.');
-} else if (daughters === 'no') {
-  //console.log('You\'re a keen observer! That\'s right, I have two.');
-  alert('You\'re a keen observer! That\'s right, I have two.');
-  score++;
-} else {
-  //console.log('You are a rebel, ain\'t ya?');
-  alert('You are a rebel, ain\'t ya?');
-}
-
-//Question 6:
-//Where does swiss cheese rank on my Top 10 List of favorite cheeses? hint: 1-10
-
-//Define number of attempts
-let attempts = 4;
+function quiz(score){
```

Comment:
Using score as an argument is a good way to think about functions, but it's unnecessary to use it as an argument. You already set score as a global variable, and your quiz function isn't going to be reused anywhere, so it doesn't need to call score as an argument.

Lets say you wrote a function that adds three, it would look something like this:
```
function addThree(value) {
  value+=3;
}
```
This is great if you're trying to add three to different things, like adding three apples to the fruit basket, and then adding three apples to the pantry. However if you're only ever going to add apples to the fruit basket, you don't need to specify fruit basket in your arguments. Your addThree function could achieve the same results written like this:
```
function addThree() {
  fruitBasket += 3;
}
```
You'll also notice that neither time I returned anything. That's because in both cases by accessing a global variable we're saying "this thing that exists outside of this function is going to be this now."